documentation/Glossary.md

```diff
@@ -26,8 +26,9 @@
 | **API Producer**                                    | Software component, inside the platform, that exposes the CAMARA API. Term for the documentation.                                                                                                                                                                                                                                                                                                                                                                         | N/A                                      | API Provider                                                  |
 | **API Consumer**                                    | Software component in the Application implemented by the Developer, that consumes the CAMARA API produced by the API Producer. Term for the documentation.                                                                                                                                                                                                                                                                                                                | N/A                                      | N/A                                                           |
 | **Suffix: Id vs Identifier**                        | 	Suffix "Id" for unique fields. E.g, ApplicationId (specific code for an Application). Suffix "Identifier" for generic fields, ProductIdentifier (generic variable to identify a family of products)                                                                                                                                                                                                                                                                      | 	e.g. ApplicationId or ProductIdentifier | 	N/A                                                          |
-
-
+| **Physical Device Identifier** | This is a unique identifier allocated to the physical device by the manufacturer. For mobile devices, this is the 15-digit International Mobile Equipment Identity (IMEI), which may be appended by the current device software version to create the 16-digit IMEISV. The physical device identifier is distinct from the mobile subsciption identifier, which is associated with the SIM rather than the device. | imei, imeisv | N/A |
```

Comment:
```suggestion
| **Physical Device Identifier** | This is a unique identifier allocated to the physical device by the manufacturer. For mobile devices, this is the 15-digit International Mobile Equipment Identity (IMEI), which may be appended by the current device software version to create the 16-digit IMEISV. The physical device identifier is distinct from the mobile subscription identifier, which is associated with the SIM rather than the device. | imei, imeisv | N/A |
```